SwanContents/swancontents/serverextension/handlers/__init__.py

```diff
@@ -1,39 +0,0 @@
-
-from notebook.utils import url_path_join
-from notebook.base.handlers import path_regex
-
-from .projects import ProjectsHandler
-from .share import ShareHandler
-from .tree import TreeHandler
-from .download import DownloadHandler
-from ..filemanager.handlers import FetchHandler, ContentsHandler
-import datetime
-
-def load_jupyter_server_extension(nb_server_app):
-    """ Used as a server extension in order to install the new handlers """
-
-    new_handlers = [(r"/cernbox", TreeHandler),
-                    (r"/cernbox%s" % path_regex, TreeHandler),
-                    (r"/projects", ProjectsHandler),
-                    (r"/projects%s" % path_regex, ProjectsHandler),
-                    (r"/share", ShareHandler),
```

Comment:
I also asked to explain in the commit message where handlers like this one go, since they are no longer here.

Reply:
They are created in a different place on the previous commit. I'm going to delete this file on the previous commit then.